Sealer/sealer.cc

```diff
@@ -615,6 +621,13 @@ void kRun(const ConnectPairsParams& params,
 				uniqueGapsClosed++;
 				if (gapsclosed % 100 == 0)
 					printLog(logStream, IntToString(gapsclosed) + " gaps closed so far\n");
+
+				if (!opt::gapfilePath.empty()) {
+					gapStream << ">" << read1.id.substr(0,read1.id.length()-2)
+						  << "_" << read2_it->second.gapStart() << "-" << read2_it->second.gapEnd()
+						  << " LN:i:" << tempSeq.length() << endl;
+					gapStream << tempSeq << endl;
```

Comment:
Change these two lines to…
```c++
<< " LN:i:" << tempSeq.length() << '\n'
<< gapStream << tempSeq << '\n';
```
`std::endl` flushes the stream's buffer, which is undesirable.

Reply:
I realize that it does, I added endl there because this file is supposed to give intermediate results, meaning they should be flushed to the file as soon as the gap is closed so that you can inspect it. But sure, I'll change it.